### DISCUSSION TITLE
How to setup a line code for BotFather to notify me when someone in particular goes live on Tiktok ?

### DISCUSSION THREAD

**Discussion by Jamie131313** [python-telegram-bot/python-telegram-bot#3499] [Q&A]
Hello you all ! I'm new to Github actually. 
I'm looking for a line code or something similar to plug into my bot to notify me in my Telegram group when someone is going live on TIKTOK. 
You think it's feasable ? Thanks a lot guys for reading me, and have a good night/day.

**Comment by Bibo-Joshi** [comment]
Please have a look at [this faq entry](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#i-want-to-handle-updates-from-an-external-service-in-addition-to-the-telegram-updates-how-do-i-do-that).

**Reply by Jamie131313** [reply]
Hey Bibo-Josi, i took a look as you said, but i didn't find any informations about my question :( 
Thanks for your answer !

**Reply by clot27** [reply]
> Hey Bibo-Josi, i took a look as you said, but i didn't find any informations about my question :( Thanks for your answer !

Do you know how to code a bot? What have you tried and that didn't worked for you,?

**Reply by Jamie131313** [reply]
@clot27 Hey, sorry i don't know anything about it... I didn't tried everything since i have a business and i'm in need of a bot who can put Tiktok live alerts on my Telegram groups. But i don't really know anything about it.

**Reply by Bibo-Joshi** [reply]
Then please get started [here](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API).
